.github/workflows/iroha2-add-label.yml

```diff
@@ -1,7 +1,7 @@
 name: Add Iroha2 label
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
It will not run without `pull_request_target` as it is the only event that allows to edit the PR AFAIK

Reply:
https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request_target